cmake/h5z_zfp-config.cmake.in

```diff
@@ -0,0 +1,59 @@
+# h5z_zfp-config.cmake
+# --------------------
+#
+# Finds the H5Z_ZFP library, specify the starting search path in H5Z_ZFP_ROOT
+#
+# Static vs. shared
+# ----------------- 
+# To make use of the static library instead of the shared one, one needs
+# to set the variable H5Z_ZFP_USE_STATIC_LIBS to ON before calling find_package.
```

Comment:
I just want to make sure that you understand that in the case of a *static* build, an application is then required to *explicitly* link to the filter plugin code. The HDF5 library *finds* the plugin code in the executable of the application itself because the application gives it a pointer to that code. The HDF5 library will not *find* the plugin via the `HDF5_PLUGIN_PATH` env. variable and dynamically load it.

Reply:
Yes, I am aware of that. In my current project we make use of the _static_ build and therefore we explicit link to the static version of the filter.